tests/drivers/spi/spi_loopback/overlay-mcux-dspi-dma.conf

```diff
@@ -3,4 +3,4 @@ CONFIG_DMA=y
 CONFIG_DSPI_MCUX_EDMA=y
 CONFIG_MCUX_DSPI_BUFFER_SIZE=144
 CONFIG_SPI_LOOPBACK_FAST_FREQ=1000000
-CONFIG_DMA_MCUX_USE_DTCM_FOR_DMA_DESCRIPTORS=n
```

Comment:
Was planning on squashing all the fixup commits into a single commit before merging to avoid the commit-guidelines rules, but leaving them as is for reviewers for now.

Have squashed and also addressed I think all of the other compliance issues if you don't mind kicking off the compliance check again @danieldegrasse . Thanks for noticing!